src/Extensions/Diagnostics.HealthChecks/HealthChecksBuilderExtensions.cs

```diff
@@ -40,6 +41,7 @@ public static class HealthChecksBuilderExtensions
 			string? scheme = null,
 			IReadOnlyDictionary<string, IEnumerable<string>>? headers = null,
 			HttpStatusCode? expectedStatus = null,
+			HealthStatus failureStatus = HealthStatus.Unhealthy,
```

Comment:
failureStatus should be after name to align with other registration APIs, (https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.healthchecksbuilderaddcheckextensions.addtypeactivatedcheck?view=dotnet-plat-ext-6.0)

Reply:
Do you mean making type nullable? I've changed it to be similar to `AddTypeActivatedCheck`, please have a look once more.